### DISCUSSION TITLE
Error while using MySQL as a connector on airbyte locally

### DISCUSSION THREAD

**Discussion by techjahnavi** [airbytehq/airbyte#34435] [Connector Questions] [answered]
Hi, 

I'm facing the following error while configuring Airbyte as a connector. Kindly help me to resolve the error. I have attached the stack trace and logs with relevant screenshots. 

Thanks, 
Jahnavi

![Screenshot 2024-01-23 122348](https://github.com/airbytehq/airbyte/assets/83537545/32e661d1-be5f-482f-a01f-1b51030ef54e)
[check_connection_source-failure-8033dd65-d5eb-4ccf-a1d7-00a19ce215b9.txt](https://github.com/airbytehq/airbyte/files/14019471/check_connection_source-failure-8033dd65-d5eb-4ccf-a1d7-00a19ce215b9.txt)

**Comment by sajarin** [comment]
hey @techjahnavi, can you double check the credentials for this connection, I believe this is failing on the check step.

**Reply by techjahnavi** [reply, marked as answer]
Hi @sajarin , 
I have debugged this issue. The credentials were correct, but in the host section, it was expecting the IP address of my machine.
